src/bootstrap/config.toml.example

```diff
@@ -152,6 +152,9 @@
 # known-good version of OpenSSL, compile it, and link it to Cargo.
 #openssl-static = false
 
+# Run the build with low priority
```

Comment:
Could you expand this to explain a bit about what this means? Referencing `nice` here for Unix and "low priority" job object for Windows would probably be sufficient.